### PR TITLE
Rename Amendment -> Modification

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

File: alcs-api/src/board/board.controller.ts [hunk 1]
```diff
@@ -60,9 +60,9 @@ export class BoardController {
       planningReviews = await this.planningReviewService.getCards();
     }
 
-    let amendments: ApplicationAmendment[] = [];
+    let modifications: ApplicationModification[] = [];
     if (boardCode === 'ceo') {
```
Comment (mhuseinov) [inline alcs-api/src/board/board.controller.ts:64]: We need gradually get rid of hardcoded values and switch to enums.

File: alcs-api/src/common/automapper/reconsideration.automapper.profile.ts [hunk 2]
```diff
@@ -88,6 +88,10 @@ export class ReconsiderationProfile extends AutomapperProfile {
           (a) => a.card,
           mapFrom((rd) => this.mapper.map(rd.card, Card, CardDto)),
         ),
+        forMember(
```
Comment (mhuseinov) [inline alcs-api/src/common/automapper/reconsideration.automapper.profile.ts:91]: Do you really need this one? Docs mentions that automapper is able to understand boolean | null

File: alcs-frontend/src/app/features/application/decision/decision-dialog/decision-dialog.component.ts [hunk 3]
```diff
@@ -246,7 +246,7 @@ export class DecisionDialogComponent implements OnInit {
   }
 
   onSelectPostDecision(postDecision: { type: string }) {
-    if (postDecision.type === 'amendment') {
+    if (postDecision.type === 'modification') {
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/application/decision/decision-dialog/decision-dialog.component.ts:249]: maybe an enum for this?

File: alcs-frontend/src/app/features/application/decision/decision.component.scss [hunk 4]
```diff
@@ -46,7 +46,7 @@ section {
     background: $accent-color;
   }
 
-  &.amendment {
+  &.modification {
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/application/decision/decision.component.scss:49]: 👍

**Review by mhuseinov — APPROVED** [review]

File: alcs-api/src/board/board.controller.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-api/src/board/board.controller.ts:64]: Added enum for board codes

File: alcs-frontend/src/app/features/application/decision/decision-dialog/decision-dialog.component.ts (hunk 3 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/application/decision/decision-dialog/decision-dialog.component.ts:249]: Good call, cleaned up

File: alcs-api/src/common/automapper/reconsideration.automapper.profile.ts (hunk 2 above)
Comment (dhaselhan) [reply alcs-api/src/common/automapper/reconsideration.automapper.profile.ts:91]: Yes, without this it is no longer mapping and doesnt return the property at all.

**Pull request merged**